client/src/models/Person.js

```diff
@@ -329,18 +329,23 @@ export default class Person extends Model {
     if (this.position && this.position.type !== Position.TYPE.SUPER_USER) {
       return false
     }
-    if (org.type === Organization.TYPE.PRINCIPAL_ORG) {
-      return true
-    }
```

Comment:
The server-side allows superusers to e.g. edit principal positions (see the calls to `AuthUtils.assertSuperUserForOrg()` in PositionResource.java) or create/edit principal persons (see the call to `AuthUtils.isSuperUserForOrg()` in PersonResource.java); is this still handled correctly on the client-side?